assets/img/va-primary-horizontal-black-vector.svg

```diff
@@ -0,0 +1,9645 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="321px" height="74px" viewBox="0 0 321 74" focusable="false" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 43.2 (39069) - http://www.bohemiancoding.com/sketch -->
+    <title>va-primary-horizontal-black-vector</title>
```

Comment:
The `<title>` and `<desc>` will need to be removed to make the SVG work with screen readers. VoiceOver wants to announce the title, which we want to prevent.